.nix/mmr-polkadot-bin.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, rust-nightly }:
+with pkgs;
+rustPlatform.buildRustPackage rec {
+  # HACK: break the nix sandbox so we can build the runtimes. This
+  # requires Nix to have `sandbox = relaxed` in its config.
+  # We don't really care because polkadot is only used for local devnet.
+  __noChroot = true;
+  name = "mmr-polkadot-v${version}";
+  version = "0.9.27";
+  src = fetchFromGitHub {
+    repo = "polkadot";
```

Comment:
please make reuse of this stuff. share it with usual polkadot and replace hashes via nix. version, rev, hash, cargoSha256, name could be inputs to template.

Reply:
done.